src/gt4py/next/program_processors/codegens/gtfn/gtfn_module.py

```diff
@@ -150,9 +148,7 @@ def __call__(
         )
 
         # combine into a format that is aligned with what the backend expects
-        parameters: list[
-            interface.ScalarParameter | interface.BufferParameter | interface.ConnectivityParameter
-        ] = [*regular_parameters, *connectivity_parameters]
+        parameters: list[interface.Parameter] = [*regular_parameters, *connectivity_parameters]
```

Comment:
Both arguments seem to be lists (even if there are missing return type hints) so I would do this and fix the missing return type hints:
```suggestion
        parameters: list[interface.Parameter] = regular_parameters + connectivity_parameters
```

Reply:
done